ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionViewModel.kt

```diff
@@ -140,6 +140,8 @@ internal constructor(
 
   lateinit var submissionId: String
 
+  var isNavigatingUp = false
```

Comment:
Can this be moved to the fragment as well and made private? I don't think it is needed outside of the fragment.